index.php

```diff
@@ -1396,4 +1396,85 @@ function wooyellowcube_init()
     $wooyellowcube = new WooYellowCube();
 }
 
+/**
+ * Install callback for the WooYellowcube plugin.
+ *
+ * Creates necessary database tables.
+ *
+ * @since 2.5.1
+ */
+function wooyellowcube_install()
+{
+    require_once( ABSPATH . 'wp-admin/includes/upgrade.php' );
+    global $wpdb;
+    $charset_collate = $wpdb->get_charset_collate();
+
+    // TODO prefix?!?!?!
```

Comment:
Adding prefixes would probably be the right thing to do, but would require:
- data migration
- code updates everywhere where tables are queried